src/main/java/org/jabref/gui/groups/GroupTreeViewModel.java

```diff
@@ -260,9 +260,60 @@ boolean onlyMinorChanges(AbstractGroup oldGroup, AbstractGroup newGroup) {
         return true;
     }
 
+    /**
+     * Opens "Rename Group Dialog" and change name of group
+     */
+
+    public void renameGroup(GroupNodeViewModel oldGroup) {
+        currentDatabase.ifPresent(database -> {
+            AbstractGroup oldGroupDef = oldGroup.getGroupNode().getGroup();
+            String oldGroupName = oldGroupDef.getName(); // Zachytenie starého názvu pred otvorením dialógu
+
+            Optional<AbstractGroup> newGroup = dialogService.showCustomDialogAndWait(
+                    new RenameGroupView(database,
+                            oldGroup.getGroupNode().getGroup())
+            );
+
+            newGroup.ifPresent(group -> {
+
+                String newGroupName = group.getName();
+
+                if (newGroupName.trim().isEmpty()) {
+                    return;
+                }
+
+                // check if is not include ","
```

Comment:
Remove comment - the next line says the same.

The WHY is important. Why is this check done.

Reply:
Hello. I removed this comment. I check this because I inspired by Edit group where this check too. I can remove this if-statement if you want.